src/App.js

```diff
@@ -24,13 +26,69 @@ function App() {
     zoom: 11,
   });
 
+  function getCoordinates(pos) {
+    const crd = pos.coords;
+    let latitude = crd.latitude;
+    let longitude = crd.longitude;
+    setMapProperties({
+      ...mapProperties,
+      center: {
+        lat: latitude,
+        lng: longitude,
+      },
+    });
+    setUserHasPanned(false); // ensures MapCenter dot not displayed when map first moves to user location
+  }
+
+  function logError(err) {
+    console.warn(`ERROR(${err.code}): ${err.message}`);
+  }
+
+  const handleCenterClick = (e) => {
+    const { map } = mapsObj;
+    e.preventDefault();
+    map.setCenter(mapProperties.center);
+    setUserHasPanned(false);
+  };
+
+  const handleLocationSharedClick = () => {
+    allowMarker(true);
+    const options = {
+      enableHighAccuracy: true,
+      timeout: 5000,
+      maximumAge: 0,
+    };
+
+    navigator.geolocation.getCurrentPosition(getCoordinates, logError, options);
+  };
+
+  const handleTitleClick = () => {
+    window.location.reload(false);
```

Comment:
I can understand why you took this approach, but the attached GIF shows the problem with it. Clicking in the title reloads the whole page. If I've already shared my location, I have to do so again. If I'm centered on anywhere other than the default, I'm taken back to the default. 😬 

Another approach that doesn't have that problem would be to use state to manage whether there's a pin location showing. If no location is showing, show the default info. Have the title reset the app to that state. Let me know if you have questions!

![Screen Cast 2020-06-03 at 8 17 24 PM](https://user-images.githubusercontent.com/4306/83711172-5b81b400-a5d7-11ea-8433-dfb52a902225.gif)